pkg/controller/build/buildrequest/assets/Containerfile.on-cluster-build-template

```diff
@@ -28,6 +18,25 @@ COPY --from=extract /etc/machine-config-daemon/currentconfig /etc/machine-config
 # since it should be set by the container runtime / builder.
 RUN container="oci" exec -a ignition-apply /usr/lib/dracut/modules.d/30ignition/ignition --ignore-unsupported <(cat /etc/machine-config-daemon/currentconfig | jq '.spec.config') && \
 	ostree container commit
+{{if and .ExtensionsImage .Extensions}}
+COPY --from={{.ExtensionsImage}} / /run/mco-extensions/os-extensions-content
+# Add the extensions repo to /etc/yum.repos.d/coreos-extensions.repo
+RUN cat <<EOF > /etc/yum.repos.d/coreos-extensions.repo
+[coreos-extensions]
+enabled=1
+metadata_expire=1m
+baseurl=/run/mco-extensions/os-extensions-content/usr/share/rpm-ostree/extensions/
+gpgcheck=0
+skip_if_unavailable=False
+EOF
+# Set file permissions
+RUN chmod 644 /etc/yum.repos.d/coreos-extensions.repo
+# Install the extensions via rpm-ostree
+RUN extensions="{{- range $index, $item := .Extensions }}{{- if $index }} {{ end }}{{$item}}{{- end }}" && \
+    echo "Installing packages: $extensions" && \
+    rpm-ostree install $extensions && \
+    ostree container commit
```

Comment:
suggestion: We should delete the `/etc/yum.repos.d/coreos-extensions.repo` file after installing the extensions since we shouldn't leave that config in place.

Reply:
Fixed